doc/tutorials/basic/index.md

```diff
@@ -0,0 +1,332 @@
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`note;2.5em;sd-mr-1` Develop in a Notebook
+:link: develop_notebook
+:link-type: doc
+
+Discover tips and tricks that will make you *swift as the wind* at exploring turbine data and building wind turbine data apps in a notebook.
+:::
+
+:::{grid-item-card} {octicon}`code;2.5em;sd-mr-1` Develop in an Editor
+:link: develop_editor
+:link-type: doc
+
+Discover tips and tricks that will increase your *developer capacity* from kilowatts to megawatts when developing in an editor!
+:::
+
+:::{grid-item-card} {octicon}`people;2.5em;sd-mr-1` Join the community
+:link: join_community
+:link-type: doc
+
+Find inspiration in the community forums, ask for help, help others, share your work, report issues, and add feature requests.
+:::
+
+::::
+
+## Display Content
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`zap;2.5em;sd-mr-1` Display Content with `pn.panel`
+:link: pn_panel
+:link-type: doc
+
+Display our beloved wind turbine images, data, and plots easily and dynamically with `pn.panel`.
+:::
+
+:::{grid-item-card} {octicon}`device-desktop;2.5em;sd-mr-1` Display Content with Panes
+:link: panes
+:link-type: doc
+
+Fine-tune how the wind turbine data is displayed with Panes, ensuring our data apps can meet the unique demands of wind turbine data visualization.
+:::
+
+:::{grid-item-card} {octicon}`pulse;2.5em;sd-mr-1` Display Performance with Indicators
+:link: indicators_performance
+:link-type: doc
+
+Show the performance of your wind turbines with Panel's Indicators.
+:::
+
+:::{grid-item-card} {octicon}`clock;2.5em;sd-mr-1` Build a Monitoring Dashboard
+:link: build_monitoring_dashboard
+:link-type: doc
+
+Build a periodically refreshing dashboard to monitor your wind turbines.
+:::
+
+::::
+
+## Organize Content
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`zap;2.5em;sd-mr-1` Use Templates
+:link: templates
+:link-type: doc
+
+Use pre-made templates to easily layout our apps with a header, sidebar, and main area.
+:::
+
+:::{grid-item-card} {octicon}`table;2.5em;sd-mr-1` Layout Content
+:link: layouts
+:link-type: doc
+
+Layout Panel apps as beautifully as efficiently as a wind farm is laid out.
+:::
+
+:::{grid-item-card} {octicon}`screen-full;2.5em;sd-mr-1` Control the Size
+:link: size
+:link-type: doc
+
+We will discover how sizing works in Panel, exploring the differences between inherent sizing, fixed sizing and responsive sizing. Finally we will cover responsive layouts.
+:::
+
+:::{grid-item-card} {octicon}`arrow-switch;2.5em;sd-mr-1` Align Content
+:link: align
+:link-type: doc
+
+We will align our wind turbine images perfectly using `align`, `margin` and `Spacer`s.
+:::
+
+:::{grid-item-card} {octicon}`file;2.5em;sd-mr-1` Build a Report
+:link: build_report
+:link-type: doc
+
+Construct a static Wind Turbine Report, exportable to HTML for efficient distribution to management.
+:::
+
+::::
+
+## Handle User Input
```

Comment:
@philippjfr has promised to do a first version of the `Handle User Input` section.

I think he has some thoughts about what to include including `pn.bind`, `Param`, `pn.rx` etc. that I don't have yet.